server/plugins/LFUGens.cpp

```diff
@@ -1096,13 +1087,16 @@ void VarSaw_Ctor(VarSaw* unit) {
     }
 
     unit->mFreqMul = unit->mRate->mSampleDur;
-    unit->mPhase = ZIN0(1);
-    float duty = ZIN0(2);
-    duty = unit->mDuty = sc_clip(duty, 0.001, 0.999);
+    double phase = unit->mPhase = sc_clip(static_cast<double>(ZIN0(1)), 0.0, 1.0);
```

Comment:
I wonder if we should wrap it instead, so that an initial phase of 1.5 becomes 0.5. It would produce a more similar result to what the unit does now (minus the init value glitch you fixed).

Alternatively, we should write more explicitly in schelp that the value is clipped.

Reply:
Good call, updated!